packages/forge/blocks/posthog/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "@typebot.io/posthog-block",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "exports": {
+    ".": "./src/index.ts",
+    "./schemas": "./src/schemas.ts"
+  },
+  "devDependencies": {
+    "@typebot.io/forge": "workspace:*",
+    "@typebot.io/tsconfig": "workspace:*"
+  },
+  "dependencies": {
+    "posthog-node": "^4.3.2"
```

```suggestion
    "posthog-node": "4.3.2"
```

We try to pin to specific versions on all the deps in Typebot :)

---

Updated